src/runtime/virtcontainers/network.go

```diff
@@ -1486,7 +1486,7 @@ func addTxRateLimiter(endpoint Endpoint, maxRate uint64) error {
 	var netPair *NetworkInterfacePair
 	var linkName string
 	switch ep := endpoint.(type) {
-	case *VethEndpoint, *IPVlanEndpoint, *TuntapEndpoint, *BridgedMacvlanEndpoint:
+	case *VethEndpoint, *IPVlanEndpoint, *TuntapEndpoint, *MacvlanEndpoint:
```

Comment:
@egernst ack. The interconnect model dictates how we connect. It should be macvlan and not bridgedmacvlan